app/components/common/RatingSlice.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import { View, StyleSheet } from 'react-native'
+import RNText from './RNText'
+import colors from '../../config/colors'
+
+const RatingSlice = ({ title, redValue, amberValue, greenValue }) => {
+  return (
+    <View style={styles.container}>
+      <View style={styles.titleContainer}>
+        <RNText style={styles.title}>{title}</RNText>
+      </View>
+      <View style={styles.row}>
+        <View style={styles.columnHeader}>
+          <RNText style={styles.redTitle}>Red</RNText>
+        </View>
+        <View style={styles.columnHeader}>
+          <RNText style={styles.amberTitle}>Amber</RNText>
+        </View>
+        <View style={styles.columnHeader}>
+          <RNText style={styles.greenTitle}>Green</RNText>
+        </View>
+      </View>
+      <View style={styles.breakerContainer}>
+        <View style={styles.horizontalLine} />
+        <View style={styles.horizontalLine} />
+        <View style={styles.horizontalLine} />
+      </View>
+      <View style={styles.row}>
+        <View style={styles.cell}>
+          <RNText style={styles.title}>{redValue}</RNText>
+        </View>
```

Comment:
Again you could also look at splitting this into its own component. RatingSliceCell, just helps reduce the duplicate code in this file.